### ISSUE TITLE
Treat residential lots as community lots

### ISSUE THREAD

**Issue by ZackCDLVI** [NightZack/HomeTactics#1]
Seriously, this option is going to be hard coded into the game and enabled by default with options to temporary or perma-disable via cheats. This is my biggest criticism about TS1 and TS2.

**Comment by francot514** [comment]
Hey, this project is interesting idea, but really hard like PD with the neighborhood support additions, about treat residential as comunity can be done with custom portals and house modifications, also for lotzoning files..

**Comment by ZackCDLVI** [comment]
There is only so much a mods can do before recreating the engine becomes necessary. The precursor to [OpenTTD](https://www.openttd.org), an open source remake of [Transport Tycoon Deluxe](http://en.wikipedia.org/wiki/Transport_Tycoon), was TTDPatch.

The problem with the patch was that it was "limited by the same operating system and computer architecture limitations as Transport Tycoon Deluxe and has limited control over what features of the game can be altered" - [to quote Wikipedia](http://en.wikipedia.org/wiki/OpenTTD#Previous_modifications_to_Transport_Tycoon_Deluxe). The same is true for The Sims. You admitted yourself that the limitations of The Sims made Sims Ville Roleplay impossible to maintain. HomeTactics' goal is extend the life of The Sims and expand it just like OpenTTD or [CorsixTH](https://github.com/CorsixTH).

**Comment by francot514** [comment]
Youre planning do this based from PD, you will need to wait until some stuff into sims antics and world development is completed to allow use that base engine for this project, unless you want to do all this yourself??

**Comment by ZackCDLVI** [comment]
Correct! That's why I'm only going as far as prototype. The prototype only makes use of UI.far in Complete Edition. It has the loading screen and all components for the Nhood. Once the rest of the PD's engine is finished, I'll be importing that into HomeTactics too to finish the gameplay aspects.
